specification/maps/data-plane/Search/preview/2022-09-01-preview/examples/ReverseGeocoding.json

```diff
@@ -0,0 +1,86 @@
+{
+  "parameters": {
+    "api-version": "2022-09-01-preview",
+    "query": "-122.138681,47.630358",
+    "AzureKey": "[subscription-key]"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "geoJsonData": {
+          "type": "FeatureCollection",
+          "features": [
+            {
+              "type": "Feature",
+              "properties": {
+                "address": {
+                  "countryRegion": {
+                    "name": "United States"
+                  },
+                  "adminDistricts": [
+                    {
+                      "shortName": "WA"
```

Comment:
do we want to show Admin1 or Admin2 differently?

Reply:
We put Admin1 an Admin2 in the adminDistricts in order.
We use this format in forward GC API as well.

Reply:
As our discussion, we will keep this format and a field "type" to indicate the type.